### DISCUSSION TITLE
TB 129 support

### DISCUSSION THREAD

**Discussion by bam80** [kewisch/gdata-provider#780] [Support]
Hi,
I occasionally updated to TB 129.0b6 as I'm on beta update channel, and now I lost access to my Google Calendars.

Any estimate for the 129 support?
Thanks.

**Comment by kewisch** [comment]
I bumped compatibility to 129, it seems to be working fine without changes. You should be able to use Provider version 128.0.0 on Thunderbird 129 now.

**Comment by JPRuehmann** [comment]
By changig the MAX_Version to 133.* it works for the 133.* versions too.
